composer.json

```diff
@@ -8,6 +8,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ibexa/core": "~4.6.x-dev",
+        "knplabs/knp-menu": "^3.3",
```

Comment:
Do you mind if we make it a non-mandatory dependency? You're not even using any classes, you are only calling a function in the template.

Reply:
Yes, actually this was also my initial idea.